documentation/design/proposals/shared-infrastructure.adoc

```diff
@@ -469,6 +469,126 @@ The following controllers and components must be implemented:
 * Messaging-endpoint controller - Managing messaging endpoints of namespaces
 * (Phase 2) Messaging-connector controller - Managing external connectors of namespaces
 
+=== Plan behavior
+
+Both for MessagingTenantPlan and AddressPlans - Only MessagingTenantPlan used in examples here.
+
+1. No plan - No plans defined by service admin - no plan specified by messaging tenant => no plan applied => no limits
+2. Restricted - Plans defined by service admin - plan set by messaging tenant
+3. Restricted by default - Plans defined and default defined by service admin - plan not set by messaging tenant
+
```

Comment:
I'm not sure I follow the idea behind "restricted" and "restricted by default".  Are these two modes of operation?  Can you elaborate?